packages/data/src/lookitS3.ts

```diff
@@ -68,12 +79,13 @@ export default class {
   /** Create a AWS S3 upload. */
   public async createUpload() {
     this.logRecordingEvent(`Creating video upload connection.`);
-    const createResponse = await this.s3.createMultipartUpload({
+    const command = new CreateMultipartUploadCommand({
       Bucket: this.env.bucket,
       Key: this.key,
-      ContentType: "video/webm",
+      ContentType: "video/webm", // TO DO: check browser support for type/codec and set the actual value here
     });
-    this.uploadId = createResponse.UploadId;
+    const response = await this.s3.send(command);
```

Comment:
I think we spoke about this yesterday, but to repeat...

I think this can stay `video/webm`. Webm supports vp8, vp9, and av1 video codecs.  Firefox will record only in vp8, but it will play back all three video codecs. Therefore, playback is safe for all codecs.  When choosing to record video I would say this is the order: av1, vp9, vp8.  I think Firefox will always be vp8 and Chrome will always be av1.  We can let the browser tell us what it supports.